### PR TITLE
add job opening pages backend and frontend

### PR DESCRIPTION
**Pull request by mikkelcodes**

### CODE REVIEW

File: packages/website/src/templates/career.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import PortableText from '@sanity/block-content-to-react';
+import Layout from '../layout';
+import styles from './Article.module.css';
+import { graphql } from 'gatsby';
+import { NavyIntro } from '../components/navyIntro';
+
+const Career = ({ data }) => {
+  const { sanityOpenPostionPage } = data;
+
+  console.log(sanityOpenPostionPage);
```
Comment (TheKnarf) [inline packages/website/src/templates/career.js:11]: console.log can be removed

File: packages/website/src/templates/career.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          <PortableText
+            blocks={sanityOpenPostionPage._rawJobDescription}
+            projectId="mnr37rl0"
+            dataset="production"
+          />
+        </span>
+        <div>
+          <div
+            className="lg:mt-10 mt-20"
+            dangerouslySetInnerHTML={{ __html: sanityOpenPostionPage.embed }}
```
Comment (TheKnarf) [inline packages/website/src/templates/career.js:42]: Is there no better way to do this that doesn't include a dangerouslySetInnerHTML?

Comment (mikkelcodes) [reply packages/website/src/templates/career.js:42]: I'm looking into a substitute for this

Comment (mikkelcodes) [reply packages/website/src/templates/career.js:42]: If we can only source a link, that would make it easier. However, the current embed includes both a script tag and an iframe. But I'll try to come up with something

Comment (TheKnarf) [reply packages/website/src/templates/career.js:42]: Allright, perhaps we just merg it then and look for a solution in the future. I would have guessed that perhaps Sanity had some official React package with components for embeding stuff from them, but if not then maybe this is the best solution.

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**